### PR TITLE
Create load_aggregate_nupi.sql

### PR DESCRIPTION
**Pull request by cbrianbet**

### CODE REVIEW

**Review by nobert-mumo — COMMENTED** [review]
LGTM

File: Scripts/REPORTING/load_aggregate_nupi.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+SELECT DISTINCT
+MFLCode,
+f.FacilityName,
+County,
+SubCounty,
+p.PartnerName as CTPartner,
+a.AgencyName as CTAgency,
+Gender,
+age.DATIMAgeGroup as AgeGroup,
+sum(CASE WHEN art.AgeLastVisit between 0 AND 18 THEN 1 ELSE 0 END) as number_children,
```
Comment (cbrianbet) [inline Scripts/REPORTING/load_aggregate_nupi.sql:16]: @nobert-mumo @Marymary-dev 
Some help here: Are children 0-17 or 0-18

Comment (cbrianbet) [reply Scripts/REPORTING/load_aggregate_nupi.sql:16]: Thanks ←_←

Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_nupi.sql:16]: but I think `between` is usually inclusive actually

**Pull request merged**